public/css/menFashion.css

```diff
@@ -0,0 +1,141 @@
+body{
+margin: 0px;
+padding: 0px;
+height: 100vh;
+width: 100%;
+}
+
+.container{
+  width: 1140px;
+  flex-wrap: wrap;
+
+}
+.containernav{
+  display: flex;
+  width: 1140px;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+}
+.bigContainer{
+  display: flex;
+  flex-direction: column;
+    flex-wrap: wrap;
+}
+header{
```

Comment:
use classes and don't use in Css name of tag >> mush told me that . 
@Salam-Dalloul